packages/fxa-content-server/app/styles/modules/_button-row.scss

```diff
@@ -67,15 +67,25 @@ button {
       background: $error-background-color;
     }
 
-    // 'type' is needed for selector specificity
-    &[type='submit'] {
-      &:active:enabled,
-      &:active:enabled:hover {
-        background: darken($error-background-color, 10);
+    &:enabled {
+      &:hover {
+        background: $error-background-color-hover;
+      }
+
+      &:active {
+        background: $error-background-color-active;
       }
 
-      &:hover:enabled {
-        background: darken($error-background-color, 5);
+      // 'type' is needed for selector specificity
+      &[type='submit'] {
+        &:active,
+        &:active:hover {
+          background: darken($error-background-color, 10);
+        }
+
+        &:hover {
+          background: darken($error-background-color, 5);
+        }
```

Comment:
What is this being used for? I took it out and the issue seems resolved without it!

![red-buttons](https://user-images.githubusercontent.com/13018240/88867897-5d7d8300-d1d4-11ea-89de-e3f98c388d8c.gif)

Reply:
This just affects the buttons with "type=submit". If you try with the "Delete account" button, the button will display the background in blue.

Reply:
Ah I see, it's for delete account, good catch. I wonder if the delete account button is purposefully slightly lighter than the colors we were given for the other red actions, but that's how the delete account button is in production now and the settings is in the process of a redesign now so this'll work. :)